### ISSUE TITLE
Won't install sfos 3.4.0.24

### ISSUE THREAD

**Issue by sbridger** [nodevel/harbour-defender#9]
Won't install on sfos 3.4.0.24 in storeman. 
Endless "installing package harbour-defender"

**Comment by DrYak** [comment]
It is blocked by the blacklist (patterns-sailfish-app-blacklist).

**Comment by sbridger** [comment]
> It is blocked by the blacklist (patterns-sailfish-app-blacklist).

Any idea of how to work around that?

**Comment by DrYak** [comment]
The blocklist it self is by version number, so as soon as nodevel makes and releases a version 0.4.1 or 0.5 it will accept to install.

The question to be solved is: why did version <= 0.4.0 got banned in the first place?
We should ask it into the SFOS forum and see what the official Jolla dev give as an explanation and fix that problem, so upcoming 0.4.1 doesn't get blacklisted, too.

I think I remember that several months ago, I managed to have defender crash (I think: because one of the IP-ban list source doesn't work anymore) which left the hostfile in a broken state that prevented name resolution (and thus made internet access unusable). Maybe that's the bug we need to fix so Jolla dev don't re-blacklist again future versions ?

**Comment by DrYak** [comment]
[bug report filed](https://forum.sailfishos.org/t/harbour-defender-blacklisted-in-sfos-3-4-0-24/3246)

**Comment by DrYak** [comment]
Just to keep the information in sync: it seems that harbour-defender breaks phones which have full encryption activated:

- [TJC reply 1](https://together.jolla.com/question/217403/defender-app-stops-xperia-10-from-booting/?answer=222140#post-id-222140)
- [TJC reply 2](https://together.jolla.com/question/225689/xa2-stuck-in-sony-logo-white-led-e2fsck-says-bad-magic-number/?answer=225764#post-id-225764)

It could stem from `/home/nemo/.config/harbour-defender/` being a directory in the .path systemd unit, which in turn is configured as a hard requirement for multi-users target.

Making this work on SFOS 3.4.0.24 would require reorganizing this and moving it to a common directory.

**Comment by DrYak** [comment]
@nodevel : Are you still interested in maintaining this (i.e.: shall we send pull requests to you ?), or shall we fork it ?

**Comment by peterleinchen** [comment]
Yes, it was a real serious issue and Jolla gave a sh.. about helping!

It took me really looong time to get to the ground of it.

And as nodevel did not reply and left this repo unmaintained I took over under 
https://github.com/peterleinchen/harbour-defender
